nautobot/extras/jobs.py

```diff
@@ -120,6 +121,10 @@ def __init__(self):
     def __str__(self):
         return self.name
 
+    @classmethod
+    def get_absolute_url(cls):
+        return reverse("extras:job", kwargs={"class_path": cls.class_path})
```

Comment:
Adding this now makes me worry - if we add a new URL pattern for jobs as part of #1001, will we have to break backwards compatibility for this function?

Reply:
Grrr, yeah I supposed we should hold on this until we get there and just include this consideration in that design.